packages/frontend/src/utilities.ts

```diff
@@ -70,6 +70,10 @@ export type SpeakingFeedback = {
   Pronunciation: Feedback;
 };
 
+export type Error = {
+  error: string;
+};
+
```

Comment:
Why not specify literal string errors (e.g. 'couldn't grade'), and rename to `FeedbackError` to be more specific?

This will make it easier to handle the various cases of feedback failure in the front end.